frictionless/lbnlisoqueues.py

```diff
@@ -0,0 +1,46 @@
+"""Provide datapackage details specific to the EIP Infrastructure archives."""
+
+from . import core
+from . import contributors
+from . import licenses
+
+lbnlisoqueues = {
+    "name": "raw-lbnlisoqueues",
+    "title": "Raw LBLN ISO Interconnection Queues",
+    "description":
+        "Berkeley Lab compiled and analyzed data from all seven ISOs/RTOs " 
+        "in concert with 35 non-ISO utilities, representing an estimated "
+        "85% of all U.S. electricity load. We include all 'active' projects "
+        "in these generation interconnection queues through the end of 2020, "
+        "as well as data on 'completed' and 'withdrawn' projects for five of "
+        "the ISOs (CAISO, ISO-NE, MISO, NYISO, PJM).",
+    "profile": "data-package",
+    "keywords": [
+        "lbln", "usa", "electricity", "iso", "renewables", "emissions", "utilities", "iso", "interconnection",
+    ],
+    "licenses": [licenses.us_govt, ],
```

Comment:
Are you sure this is accurate? The national labs aren't technically part of the USG, and their products may or may not be subject to the constitutional copyright exclusion.

Reply:
I'm not sure. I couldn't find a definitive license for LBNL data. I might need to email them.

Reply:
Same goes for the EIP dataset.